server/src/game/role/yer.rs

```diff
@@ -0,0 +1,138 @@
+                    true
+                );
+            }else{
+                if priority != Priority::Convert {return}
+                if self.star_passes_remaining <= 0 {return}
+
+                if target_ref.night_defense(game).can_block(AttackPower::ArmorPiercing) {
+                    actor_ref.push_night_message(game, ChatMessageVariant::YourConvertFailed);
+                    return
+                }
+
+                self.star_passes_remaining = self.star_passes_remaining.saturating_sub(1);
+
+                //role switching stuff
+                let fake_role = if let Some(RoleOptionSelection(Some(role))) = game.saved_controllers.get_controller_current_selection_role_option(
+                    ControllerID::role(actor_ref, Role::Yer, 2)
+                ){
+                    role
+                }else{Role::Yer};
+
+                actor_ref.set_night_grave_role(game, Some(fake_role));
+                
+                //convert & kill stuff
+                target_ref.set_win_condition(
+                    game,
+                    WinCondition::new_loyalist(crate::game::game_conclusion::GameConclusion::Fiends)
+                );
+                target_ref.set_night_convert_role_to(game, Some(RoleState::Yer(self.clone())));
+
+                actor_ref.try_night_kill_single_attacker(
+                    actor_ref,
+                    game,
+                    GraveKiller::Role(Role::Yer),
+                    AttackPower::ProtectionPiercing,
+                    true
+                );
+
+                actor_ref.set_role_state(game, self);
+            }
+        }
+    }
+    fn controller_parameters_map(self, game: &Game, actor_ref: PlayerReference) -> ControllerParametersMap {
+        crate::game::role::common_role::controller_parameters_map_boolean(
+            game,
+            actor_ref,
+            self.star_passes_remaining <= 0 || game.day_number() <= 1,
+            ControllerID::role(actor_ref, Role::Yer, 0)
+        ).combine_overwrite_owned(
+            crate::game::role::common_role::controller_parameters_map_player_list_night_typical(
+                game,
+                actor_ref,
+                false,
+                game.day_number() <= 1,
+                ControllerID::role(actor_ref, Role::Yer, 1)
+            )
+        ).combine_overwrite_owned(
+            ControllerParametersMap::new_controller_fast(
+                game,
+                ControllerID::role(actor_ref, Role::Yer, 2),
+                AvailableAbilitySelection::new_role_option(
+                    Role::values().into_iter()
+                        .map(|role| Some(role))
+                        .collect()
+                ),
+                AbilitySelection::new_role_option(Some(Role::Yer)),
+                self.star_passes_remaining <= 0 || !actor_ref.alive(game) || game.day_number() <= 1,
+                None,
+                false,
+                vec_set!(actor_ref)
```

Comment:
This always shows I think, even when you haven't chosen to convert. It's fine if this is intentional